### DISCUSSION TITLE
Selecting Playlists - Reverting to default

### DISCUSSION THREAD

**Discussion by Kishiioo** [Geektoolkit/Dynaframe3#138] [Q&A]
Hi Guys,

Amateur pi user here. I have uploaded some pictures via the Web Interface and now when I unselect the default photo directory to just view just the uploaded ones and click save. It doesn't view them and reverts to both being selected?

Any reason for this? 

Thanks,

**Comment by Geektoolkit** [comment]
What build are you on? If it's pretty recent it may be damaged as part of the rewrites I'm working on. I'll test it in 2.24 next chance I get

**Comment by Kishiioo** [comment]
Hi Joe,

I have been jumping around. Some work with some features and others don't
work at all. Just want the playlists and uploads working on 1 version. Keep
up the great work.

Thanks,
J.
